File: src/main/java/com/example/minitest/model/Tour.java
package com.example.minitest.model;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
public class Tour {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nameTour;
    private double priceTour;
    private String descriptionTour;

}

File: src/app/tour-componet/model/tour.ts

export class Tour{
  id!: number;
  nameTour!: String;
  priceTour!: number;
  descriptionTour!: String;


  constructor(id: number, nameTour: String, priceTour: number, descriptionTour: String) {
    this.id = id;
    this.nameTour = nameTour;
    this.priceTour = priceTour;
    this.descriptionTour = descriptionTour;
  }


}
